gulpfile.js

```diff
@@ -109,7 +110,7 @@ gulp.task('build:testjs', function () {
   gulp.src('src/*.js')
     .pipe(replace('{{apiurl}}', TESTAPIURL))
     .pipe(replace('{{cssurl}}', TESTCSSURL))
-    .pipe(replace('{{templates}}', prepareTemplates()))
+    .pipe(replace('\'{{templates}}\'', prepareTemplates()))
```

Comment:
Yeah, I chose to lint the source, cause otherwise the line numbers are off on the linter output, which kind of makes it a pain to hunt down what actually needs to be changed. D: